common/static/common/js/spec_helpers/ajax_helpers.js

```diff
@@ -72,6 +72,10 @@ define(['sinon', 'underscore', 'URI'], function(sinon, _, URI) {
         expect(request.readyState).toEqual(XML_HTTP_READY_STATES.OPENED);
         expect(request.url).toEqual(url);
         expect(request.method).toEqual(method);
+        if (typeof body === 'undefined') {
+            // The contents if this call may not be germane to the current test.
```

Comment:
Nit: I'm not really understanding the point of the comment here -- can you add some clarification?